python/Ganga/new_tests/GPI/GangaList/TestCopy.py

```diff
@@ -43,7 +36,7 @@ def testCopy(self):
         gl2 = copy.copy(gl)
         assert len(gl2) == len(gl), 'lists must be equal'
         assert gl2 is not gl, 'list must be copies'
-        #assert gl[0] is gl2[0], 'the references must be copied' ## rcurrie 09/12/15 Not sure this is valid given we have to cope with GangaObjects here which may not be copyable!
+        assert gl[0] is gl2[0], 'the references must be copied'
```

Comment:
I don't expect that any user will ever do a `copy.copy` of any of our objects. I agree it will cause problems with parenting and perhaps on the proxy-level it might be worth prohibiting it and only allowing `deepcopy`. Most users will simply use our `.copy()` method we provide on proxy objects.